lib/cl/framed-transport.lisp

```diff
@@ -0,0 +1,136 @@
+(in-package #:org.apache.thrift.implementation)
+
+;;;; Copyright 2017 Rigetti Computing <rigetti.com>
```

Comment:
@jfarrell do we need to check with them to make sure this is okay to submit?

Reply:
Our work on Thrift is sponsored by Rigetti, so there is no need for that.

Reply:
> Our work on Thrift is sponsored by Rigetti, so there is no need for that.

That sounds ... strange.

Reply:
Rigetti hired us to implement (and possibly merge) CL support to Thrift. All software under this delivery has to have license required by the project we are contributing to (that is Thrift). Not sure what is strange about that.

Reply:
Well, the term "sponsored" is a bit broad to derive that particular conclusion from it. So I asked for further clarification.

Reply:
@dkochmanski thank you for working on this and contributing it back the the Apache Thrift. The Apache license header on this should not contain a copyright to Rigetti Computing, details available at https://www.apache.org/legal/src-headers.html#headers part 2

Reply:
Right, thank you for pointing me to the document. Last commit removes all copyrights added by us. Note though, that files in lib/cl/ are based on work by James Anderson (and have copyrights as well) and I don't think I can remove them. Should I contact him and ask whenever we can remove his copyright stings too?

Reply:
That would be great. is it possible fetch his work as a third party downloadable dependency or was it used as the basis for work you did on top of it?